### DISCUSSION TITLE
How to calculate resulting logical size of RAIDZ-2 pool?

### DISCUSSION THREAD

**Discussion by psapezhka** [openzfs/zfs#12959] [Q&A] [answered]
Hi, I'm trying to plan infrastructure and calculate the resulting amount of usable space. I have disks quantity, size in bytes, sector size. Also, I have some non-default values of ZFS parameters, like `recordsize=1m`. It would be great to have some calculator or an algorithm to calc. 

P.S. I've googled a couple of solutions, but all of them seem to me too approximate or abandoned.

**Comment by gmelikov** [comment, marked as answer]
- raidz padding https://docs.google.com/spreadsheets/d/1tf4qx1aMJp8Lo_R6gpT689wTjHv6CGVElrPqTA0w_ZY/edit?pli=1#gid=1576424058
- slop space https://openzfs.github.io/openzfs-docs/Performance%20and%20Tuning/Module%20Parameters.html?highlight=slop#spa-slop-shift

**Reply by jumbi77** [reply]
Not sure, but isn't slop space now upper bounded to max 128G ([#11023](https://github.com/openzfs/zfs/pull/11023)) ? If yes, maybe the referenced doc should be edit to "default: 3.2% with an upper limit to 128G ?)
